static/book.css

```diff
@@ -15,7 +15,6 @@ body {
 
 .book-image-container {
     width: 250px;
-    height: 350px;
```

Comment:
removed the height because the container was clipping into the image of the book